### PR TITLE
New 2.x version.

### PR DESCRIPTION
**Pull request by pixelcab**
Still, work is to be done. Clean up, documentation, and site generator updates.

### CODE REVIEW

File: bin/compile.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    print "Unable to find autoloader for Gnorm.\n";
+    exit(1);
+}
+
+/**
+ * Options:
+ * c = The json encoded config.
+ * b = Boolean telling if this is a build.
+ */
+$options = getopt('c:b::');
```
Comment (ryanobrooks) [inline bin/compile.php]: This optional "b" option does not appear to be working as expected.  Changing it to required (`b:` vs `b::`) appears to fix it.

File: bin/generateSite.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    print "Unable to find autoloader for Gnorm.\n";
+    exit(1);
+}
+
+/**
+ * Options:
+ * c = The json encoded config.
+ * b = Boolean telling if this is a build.
+ */
+$options = getopt('c:b::');
```
Comment (ryanobrooks) [inline bin/generateSite.php]: Same thing here.

File: bin/compile.php (hunk 1 above)
Comment (pixelcab) [reply bin/compile.php]: @ryanobrooks, the optional parameter identifier was changed to use the required format.

File: bin/generateSite.php (hunk 2 above)
Comment (pixelcab) [reply bin/generateSite.php]: @ryanobrooks, the optional parameter identifier was changed to use the required format.

**Review by ryanobrooks — APPROVED** [review]

**Pull request merged**